[Ready for review] README Italian translation (ciao, mondo!)

varlottaang opened the pull request:
<!--
Please complete the following sections when you submit your pull request. You are encouraged to keep this top level comment box updated as you develop and respond to reviews. Note that text within html comment tags will not be rendered.
-->
### Summary

[Translate] README file in Italian

Fixes #1335

- Italian version of README.md


### What should a reviewer concentrate their feedback on?

- [x] Proofreading 


### Acknowledging contributors

<!-- Please select the correct box -->

- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file: @varlottaang

bertinetto commented on README-Italian.md:
```diff
@@ -1,110 +1,116 @@
-_The Turing Way_ is a lightly opinionated guide to reproducible data science.
-You can read it here: <https://the-turing-way.netlify.com>
-You're currently viewing the project GitHub repository where all of the bits that make up the guide live, and where the process of writing/building the guide happens.
+_The Turing Way_ è una guida dettagliata all riproducibilià nella scienza dei dati (data science).
```
I wouldn't translate "lightly opinionated" with "guida dettagliata"

bertinetto replied:
I would not translate "data science" and leave it in the english form everywhere, especially if you are not going to translate "data scientist".

varlottaang replied:
:) "lightly opinionated" could be translated with "arguta" "una guida arguta". In the french.md it was translated with "légèrement avisé".  

About data science, I agree with you, I prefer the English version "data science", I was undecided because I've found "scienza dei dati" in many posts and websites (ex. https://www.sns.it/it/data-science), but also "la data science", "nella data science (ex. https://www.oracle.com/it/data-science/what-is-data-science.html)

bertinetto commented on README-Italian.md:
```diff
@@ -1,110 +1,116 @@
-You're currently viewing the project GitHub repository where all of the bits that make up the guide live, and where the process of writing/building the guide happens.
+_The Turing Way_ è una guida dettagliata all riproducibilià nella scienza dei dati (data science).
+Puoi leggerla qui: <https://the-turing-way.netlify.com>
+Stai attualmente esplorando il progetto nella sua repository GitHub dove sono conservati tutti i bit che compongono la guida, e dove avvienve il processo di lettura e creazione della guida stessa.
```
"Bits" qua andrebbe tradotto con "parti".
Typo per "avvienve"

bertinetto commented on README-Italian.md:
```diff
@@ -1,110 +1,116 @@
+Stai attualmente esplorando il progetto nella sua repository GitHub dove sono conservati tutti i bit che compongono la guida, e dove avvienve il processo di lettura e creazione della guida stessa.
 
-Our goal is to provide all the information that researchers need at the start of their projects to ensure that they are easy to reproduce at the end.
+Il nostro intento è quello di fornire tutte le informazioni di cui i ricercatori hanno bisogno all'avvio dei loro progetti, in modo che questi siano facili da riprodurre una volta terminati.
```
Albeit correct, this translation might lead to misunderstandings.
I would remove "una volta terminati", which preserves the meaning but makes it easier to understand.

bertinetto commented on README-Italian.md:
```diff
@@ -1,110 +1,116 @@
-This project is openly developed and any and all questions, comments and recommendations are welcome at our github repository: <https://github.com/alan-turing-institute/the-turing-way>.
+La riproducibilità della ricerca è necessaria per garantire l'affidabilità del lavoro scientifico.
+Finanziatori ed editori sempre più spesso richiedono che le pubblicazioni includano l'accesso ai dati grezzi ed al codice sorgente.
+L'obiettivo è quello di garantire che tutti i risultati siano verificabili in maniera indipendente e sviluppati nei futuri lavori.
```
The last bit of the sentence should be translated as
"E possano essere usati come punto di partenza in lavori futuri"

bertinetto commented on README-Italian.md:
```diff
@@ -1,110 +1,116 @@
+Finanziatori ed editori sempre più spesso richiedono che le pubblicazioni includano l'accesso ai dati grezzi ed al codice sorgente.
+L'obiettivo è quello di garantire che tutti i risultati siano verificabili in maniera indipendente e sviluppati nei futuri lavori.
+Tutto questo a volte è più facile a dirsi che a farsi.
+Condividere i risultati della ricerca significa comprendere la gestione dei dati, lo sviluppo del software, le scienze bibliografiche e le tecniche di integrazione continua: comptenze che non sono ben spiegate o richieste ad accademici e data scientist. Non essendo queste materie comunemente insegnate, riconosciamo che il numero di requisiti e nuove competenze richieste possano essere un problema ...
```
"Comptenze" - typo.

bertinetto commented on README-Italian.md:
```diff
@@ -1,110 +1,116 @@
+L'obiettivo è quello di garantire che tutti i risultati siano verificabili in maniera indipendente e sviluppati nei futuri lavori.
+Tutto questo a volte è più facile a dirsi che a farsi.
+Condividere i risultati della ricerca significa comprendere la gestione dei dati, lo sviluppo del software, le scienze bibliografiche e le tecniche di integrazione continua: comptenze che non sono ben spiegate o richieste ad accademici e data scientist. Non essendo queste materie comunemente insegnate, riconosciamo che il numero di requisiti e nuove competenze richieste possano essere un problema ...
+_The Turing Way_ è un manuale per aiutare gli studenti, i loro supervisori, finanziatori ed editori a rendere la scienza dei dati "impossibile da non fare", anche per coloro che non hanno mai lavorato secondo questi schemi.
```
This should be
" [...] a rendere la **riproducibilita'** in data science "troppo facile da non fare".

BrainonSilicon commented:
Hi @varlottaang. I just wanted to check and see if this was ready for review 😄  Thank you for your work on it!

varlottaang commented:
> Hi @varlottaang. I just wanted to check and see if this was ready for review 😄 Thank you for your work on it!

Hello @BrainonSilicon . Yes, I followed the suggestions and it's ready :) 
Prego (you're welcome/not at all/no worries, in Italian)

malvikasharan commented:
Awesome work @varlottaang and @bertinetto. Before merging can I ask you to do the following as well? (let me know if you would like me to do that):

- [ ] update your local branch with recent updates from the main branch
- [ ] move your translated file to the folder README-translated
- [ ] remove the test-status table. Also, remove contributor's table detail, and instead add a link to the original table (the reason is that we will update this table constantly and would like to avoid duplication)
   - see this file for example
- [ ] Add a link to the README-Italian on the top of Repository's README by extending this statement "This README.md file in also available in Dutch (README-Dutch) and Portuguese (README-Portuguese)"

varlottaang commented:
> * [ ]  update your local branch with recent updates from the main branch
> * [ ]  move your translated file to the folder README-translated
> * [ ]  remove the test-status table. Also, remove contributor's table detail, and instead add a link to the original table (the reason is that we will update this table constantly and would like to avoid duplication)
>   
>   * [see this file for example](https://github.com/alan-turing-institute/the-turing-way/blob/master/README-translated/README-Dutch.md)
> * [ ]  Add a link to the README-Italian on the top of Repository's README by extending this statement "This README.md file in also available in Dutch (README-Dutch) and Portuguese (README-Portuguese)"

Hello @malvikasharan,
I moved the file into the "translated" folder and changed the readme.md . Did I do everything right?

BrainonSilicon commented:
> > * [ ]   update your local branch with recent updates from the main branch
> > * [ ]   move your translated file to the folder README-translated
> > * [ ]   remove the test-status table. Also, remove contributor's table detail, and instead add a link to the original table (the reason is that we will update this table constantly and would like to avoid duplication)
> >   
> >   * [see this file for example](https://github.com/alan-turing-institute/the-turing-way/blob/master/README-translated/README-Dutch.md)
> > * [ ]   Add a link to the README-Italian on the top of Repository's README by extending this statement "This README.md file in also available in Dutch (README-Dutch) and Portuguese (README-Portuguese)"
> 
> Hello @malvikasharan,
> I moved the file into the "translated" folder and changed the readme.md . Did I do everything right?

Yes! Thank you so much @varlottaang. 

Thank you for your work on this, it's amazing to have this translation as Open Science does not mean "open but only in one language". This is really great work. Happy to accept it and welcome to the community!  🎉 🌟 🎉

malvikasharan commented:
Great work everyone! This is our 4th translated README this month. 🎊 All thanks to wonderful folks like you that we can continue to make our project accessible for wider community. 🌻
